projects/Mallard/ios/Podfile.lock

```diff
@@ -322,6 +322,6 @@ SPEC CHECKSUMS:
   SSZipArchive: fa16b8cc4cdeceb698e5e5d9f67e9558532fbf23
   yoga: c2c050f6ae6e222534760cc82f559b89214b67e2
 
-PODFILE CHECKSUM: 788719fce78dff3b2ae21dad8bb5cde3c56a97bb
+PODFILE CHECKSUM: a9367795dd9fc067665dbf01f59331b6bb44d791
```

Comment:
did you mean to commit this? i don't see any other related pods changes. it is probably safer to leave this out.